docs/supported/index.rst

```diff
@@ -0,0 +1,267 @@
+========================
+Supported Configurations
+========================
+
+Supported Cluster Managers
+==========================
+
+.. raw:: html
+
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
+			<img src="/_images/docker2x.png" alt="Docker logo"/>
+			<span>Docker, Swarm, Compose <em>Fastest</em></span>
+	        <a href="docker-integration/" class="button button--fast">Learn more</a>
+	    </div>
+	    
+	    <div class="pod-boxout pod-boxout--orchestration">
+			<img src="/_images/kubernetes2x.png" alt="Kubernetes logo"/>
+			<span>Kubernetes</span>
+	        <a href="kubernetes-integration/" class="button">Learn more</a>
+	    </div>
+	    
+	    <div class="pod-boxout pod-boxout--orchestration">
+			<img src="/_images/mesos2x.png" alt="mesos logo"/>
+			<span>Mesos</span>
+	        <a href="mesos-integration/" class="button">Learn more</a>
+	    </div>
+	</div>
+	
+	 <div class="pod-boxout pod-boxout--minor pod-boxout--orchestration">
+		<span><img src="/_images/icon-question2x.png" aria-hidden="true" alt=""/>&nbsp; You can install Flocker without a specific cluster manager, click here to learn more:</span>
+        <a href="/flocker-standalone/" class="button">Learn more</a>
+    </div>
+
+.. _storage-backends:
+
+Supported Infrastructure & Storage
+==================================
+
+The sections below list the supported infrastructure and storage options.
+
+**Community Developed** drivers are those provided and supported by our community partners.
+Other options that are still be worked on by ClusterHQ are tagged with **Experimental** or **Coming Soon**:
+
+.. raw:: html
+
+    <img src="/_images/3rd-party.png" style="height:25px; margin:2px; border:1px solid #ddd;" title="Community Developed"> = Community Developed
+    <img src="/_images/experimental.png" style="height:25px; padding:2px 4px; margin-left:20px; border:1px solid #ddd;" title="Experimental (labs project)"> = Experimental
+    <img src="/_images/coming-soon.png" style="height:25px; margin:2px; margin-left:20px; border:1px solid #ddd;" title="Coming Soon"> = Coming Soon
+    <div style="clear:both; margin-top:20px;"></div>
+
+IaaS Block Storage
+------------------
+
+These are the best options for running Flocker on a supported public or private cloud.
+
+.. raw:: html
+
+    <div class="big-box storage">
+        AWS EBS
+		<img src="/_images/AWS.png" style="width:120px;"/>
+        <p style="margin-top:10px"><a href="index.html">Works with everything</a></p>
```

Comment:
These tags ("Works with everything", "works with manual installation", "works with Labs Installer") could be confusing to the end user. An end user would be interested in knowing whether Flocker can be installed on his/her OS of choice, irrespective of which installer (s)he chose to get there. Can you please consider removing these tags? Thanks!